bebot/src/bebot.cpp

```diff
@@ -0,0 +1,47 @@
+#include <bebot/bebot.hpp>
+#include <Eigen/Dense>
+#include <iostream>
```

Comment:
no IO calls in this file, and there probably shouldn't be direct IO calls. Logging functions if necessary.